cmd/handler-api.go

```diff
@@ -246,10 +246,12 @@ func maxClients(f http.HandlerFunc) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		globalHTTPStats.incS3RequestsIncoming()
 
-		if val := globalServiceFreeze.Load(); val != nil {
-			if unlock, ok := val.(chan struct{}); ok && unlock != nil {
-				// Wait until unfrozen.
-				<-unlock
+		if r.Header.Get(globalObjectPerfUserMetadata) == "" {
```

Comment:
```suggestion
		if r.Header.Get(globalObjectPerfUserMetadata) == "true" {
```

Reply:
no @poornas if it's `true` then we should by pass the throttling done in the `if` condition